@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -40,15 +44,41 @@ const Console = withState<State, Props, Effects, Computed, ParentState, ParentEf
     initialState: () => ({
       container: React.createRef(),
       rfb: undefined,
+      rfbConnected: false,
     }),
     effects: {
-      initialize: async function () {
+      initialize: function () {
+        this.effects._connect()
+      },
+      _handleDisconnect: function () {
+        this.state.rfbConnected = false
+        let tries = 0
+        const intVal = setInterval(() => {
+          if (this.state.rfbConnected) {
+            clearInterval(intVal)
+            return
+          }
+          if (++tries === 10) {
+            clearInterval(intVal)
+            throw new Error('Unable to connect to the VM console. Too much attempts')
+          }
+          this.effects._connect().catch(({ message }) => {
+            if (message === 'Could not find VM console' || message === 'Not connected to XAPI') clearInterval(intVal)
```

Comment:
Don't we want to try to reconnect anyway?

Reply:
For me, it have no sense to try to reconnect in these error case.
For exemple, if the `VM` is offline, we will have an error message `Could not find VM console`. It's not really usefull to try to connect to VM console if this one is onffline no ?